crates/net/network/src/manager.rs

```diff
@@ -916,7 +916,7 @@ pub enum NetworkEvent {
         /// A request channel to the session task.
         messages: PeerRequestSender,
         /// The status of the peer to which a session was established.
-        status: Status,
+        status: Arc<Status>,
```

Comment:
we can do that because we're not updating any fields once the connection has been established and since block propagation is no longer a thing there's no need to update the block anyway and we also don't touch the forkid afterwards

Reply:
Yeah, can change to Box in the future if necessary since it's the same API